pyproject.toml

```
@@ -26,9 +26,9 @@ albumentations = "^1.0.0"
 ethicml = "^0.3.0"
 lightning-bolts = "^0.3.4"
 palkit = {git = "https://github.com/predictive-analytics-lab/palkit.git", branch = "main" }
-torch = "*"
-torchvision = "*"
-torchaudio = "*"
+torch = "^1.8"
```

torchvision 0.10.1 needs torch 1.9.1 as far as I know. So you might as well specify it like that.

```suggestion
torch = "^1.9.1"
```